packages/dataviews/src/components/dataviews-filters/style.scss

```diff
@@ -317,7 +317,7 @@
 	.dataviews-filters__search-widget-filter-combobox__input {
 		@include input-control;
 		display: block;
-		padding: 0 $grid-unit-40 0 $grid-unit-10;
+		padding: 0 $grid-unit-10 0 $grid-unit-40;
```

Comment:
To test, go to a DataViews story and add a "Type" filter.

<img width="678" height="472" alt="Filter widget in DataViews" src="https://github.com/user-attachments/assets/19072480-7e7a-4cbf-8a77-da2de6e997cc" />